src/internal.c

```diff
@@ -33618,8 +33642,14 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
             if (b == ID_LEN)
                 ssl->options.resuming = 1; /* client wants to resume */
             WOLFSSL_MSG("Client wants to resume session");
-            i += b;
         }
+#ifdef HAVE_SECURE_RENEGOTIATION
+        else {
+            /* We don't want to resume in SCR */
```

Comment:
NIT: consider consolidating this sessionIdSz reset with resuming reset on line 33436.

Reply:
I'm not sure what you want me to do here?

Reply:
If you set `sessionIdSz = 0 ` right after 
```
/* Reset to sane value for SCR */
        ssl->options.resuming = 0;
```
you can avoid this guarded else branch.
It's not blocking for me so marking it as resolved,  feel free to apply it or not